<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>642</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>642</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Memorize</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/cards/f</normaloff>:/cards/f</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>654</width>
      <height>584</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Are You Ready To Memorize!?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_16">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/cards/f</normaloff>:/cards/f</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>512</width>
           <height>512</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="accessibleName">
            <string/>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>4</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>8</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_12">
           <property name="text">
            <string>12</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_1">
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>5</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>9</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_13">
           <property name="text">
            <string>13</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>6</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>10</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_14">
           <property name="text">
            <string>14</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>7</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_11">
           <property name="text">
            <string>11</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_15">
           <property name="text">
            <string>15</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/cards/b</normaloff>:/cards/b</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>160</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>First Card</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Second Card</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>First Card #</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_3">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Second Card #</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_4">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>15</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>631</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>621</width>
        <height>581</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;How To Play!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;At the start of the game a buzzer will sound and the cards will be shuffled. The game will prompt you to &amp;quot;Memorize!&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Memorize as many cards as you can before closing the window!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Once you think you have the cards memorized, close the window and the cards will be flipped over.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Once you have successfully matched all the cards, you will be declared the winner!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;*Choose File-&amp;gt;New from the menu at any time to start a new game!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;**Choose File-&amp;gt;Help from the menu at any time to view this help window again!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionHelp"/>
    <addaction name="actionCheck"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
